pkg/controller/controller.go

```diff
@@ -102,8 +105,36 @@ func NewController(clientset kubernetes.Interface, ssclientset ssclientset.Inter
 		},
 	})
 
+	sInformer := sinformer.Core().V1().Secrets().Informer()
+	sInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
+		DeleteFunc: func(obj interface{}) {
+			key, err := cache.MetaNamespaceKeyFunc(obj)
+			if err != nil {
+				return
```

Comment:
We should log the error at the very least.

```suggestion
			if err != nil {
			        log.Printf("failed to fetch key: %v", err)
				return
```

Otherwise we expose ourselves to long debugging hours for no good reason.

Same for the other returns.

Reply:
Let me check the messages. I am going to review the other returns but several of them are used only to check that the Secret has a Sealed Secret. Include messages to those parts are going to generate more noise than help.

Reply:
I agree with @alvneiayu. This code is checking if a Secret is managed by a certain SealedSecret, I don't think we should polute the logs with messages about Secrets not managed by a Sealed Secret.

Reply:
I am totally fine with skipping expected  error results, such as `not found`. But we should still log if the error was something else we did not expect to happen.